### ISSUE TITLE
Make uploads simultaneous

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#3566]
When on wifi, it might be good to have multiple uploads in parallel, like the iOS app.
We shall inspect how and if this is possible, also in relation with Doze, ect.

**Comment by Gatak** [comment]
I'd this could scale to parallel download of thumbnails it would be great :)

**Comment by szaimen** [comment]
Duplicate of #6794
(Actually is #6794 a duplicate of this issue but there is already more feedback. Hence closing this one)
